spec/yaml/swiftnav/sbp/imu.yaml

```diff
@@ -0,0 +1,55 @@
+# Copyright (C) 2016 Swift Navigation Inc.
+# Contact: Fergus Noble <fergus@swiftnav.com>
+#
+# This source is subject to the license found in the file 'LICENSE' which must
+# be be distributed together with this source. All other rights reserved.
+#
+# THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+# EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+
+package: swiftnav.sbp.imu
+description: Inertial Measurement Unit (IMU) messages.
+stable: False
+public: True
+include:
+  - types.yaml
+definitions:
+
+ - MSG_IMU_RAW:
+    id: 0x0900
+    short_desc: Raw IMU data
+    desc: |
+      Raw data from the Inertial Measurement Unit, containing accelerometer and
+      gyroscope readings.
+    fields:
+        - tow:
```

Comment:
Other things that come to mind for a flag: whether or not the IMU measurement is calibrated.